### ISSUE TITLE
Kotlin example not working

### ISSUE THREAD

**Issue by Maurice-Mueller** [micronaut-projects/micronaut-examples#24]
I just tried the java and kotlin example. Java is working, kotlin is not. For both I used gradle and tested with both JDK 8 and JDK 10.

I cannot figure out the problem.

Edit: Netty is up and running and there are no compilation or runtime errors.

**Comment by graemerocher** [comment]
Not working in what sense?

**Comment by Maurice-Mueller** [comment]
In the sense that I get no responses on localhost (http://localhost:8080/hello/John). I get `{"_links":{"self":{"href":"/hello/John","templated":false}},"message":"Page Not Found"}` .

I just had a deeper look into this and it only happens with JDK 10.

To reproduce:

- clone project
- cd into hello-world-kotlin
- set JDK to 10
- `./gradlew run`  => error as described above happens

Previously I wrote that it does not work for JDK 8 as well. This is not true but was due to the fact that gradle didn't recompile the files when I first changed from JDK 10 to JDK 8 for testing. If I remove the `build` folder and run `./gradlew run` with JDK 8 everything works fine.

**After another search I finnally found the error: the kotlin version needs to be updated - then it runs for JDK 10 as well. I opened a small pull request.**
